app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt

```diff
@@ -605,7 +605,7 @@ class RecentlyPlayedFragmentTest {
       ItemSelectionInputModule::class, MultipleChoiceInputModule::class,
       NumberWithUnitsRuleModule::class, NumericInputRuleModule::class, TextInputRuleModule::class,
       DragDropSortInputModule::class, ImageClickInputModule::class, InteractionsModule::class,
-      GcsResourceModule::class, GlideImageLoaderModule::class, ImageParsingModule::class,
+      GcsResourceModule::class, TestImageLoaderModule::class, ImageParsingModule::class,
```

Comment:
as we are mentioning all the modules in all the test files,  we should add this module `TestImageLoaderModule` to each test files. 

This does sound a lot of repetitive work, @BenHenning either we can have a central utility if we need all module for each test file where we can pass every class to inject like `fun inject(recentlyPlayedFragmentTest: RecentlyPlayedFragmentTest)` 
or should I have a good first issue to track which file needs which module and remove unwanted modules from all test files?

Reply:
I think we should add test modules as needed on a test suite basis until #1720 is resolved (since that should automatically generate the test application component for us).